index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Tacettin' Portfolio</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      crossorigin="anonymous"
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      crossorigin="anonymous"
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <nav>
+      <a href="#" id="my-logo">My Logo</a>
+      <a title="menu" href="#" id="hamburger-menu">
+        <i class="fa fa-bars"></i>
+      </a>
+    </nav>
+    <section>
+      <h2 id="first-section">I’m Mariane Glad to see you!</h2>
+      <p id="second-section">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <p id="connect">LET’S CONNECT</p>
+      <div id="social-media">
+        <img src="./utils/Vector.svg" alt="elipse" id="twitter" />
+        <img src="./utils/Vector.png" alt="elipse" id="linkedin" />
+        <img src="./utils/Vector1.png" alt="elipse" id="linkedin1" />
+        <img src="./utils/Vector2.png" alt="elipse" id="linkedin2" />
+        <img src="./utils/Vector1.svg" alt="elipse" id="medium" />
+        <img src="./utils/Vector2.svg" alt="elipse" id="github" />
+        <img src="./utils/Vector3.svg" alt="elipse" id="angel" />
+      </div>
+    </section>
+    <figure>
+      <img src="./utils/Ellipse8.svg" alt="elipse" id="ellipse" />
+      <img src="./utils/Rectangle6.png" alt="rectangle" id="rectangle" />
+      <img src="./utils/Ellipse11.png" alt="rectangle" id="ellipse11" />
+      <img src="./utils/Rectangle7.png" alt="rectangle" id="rectangle7" />
+      <img src="./utils/Ellipse5.png" alt="rectangle" id="ellipse5" />
+      <img src="./utils/Rectangle4.png" alt="rectangle" id="rectangle4" />
+      <img src="./utils/Ellipse3.svg" alt="rectangle" id="ellipse3" />
+      <img src="./utils/Ellipse14.png" alt="rectangle" id="ellipse14" />
+      <img src="./utils/Ellipse12.png" alt="rectangle" id="ellipse12" />
+      <img src="./utils/Ellipse13.png" alt="rectangle" id="ellipse13" />
+      <img src="./utils/Ellipse4.png" alt="rectangle" id="ellipse4" />
+      <img src="./utils/Rectangle5.png" alt="rectangle" id="rectangle5" />
+      <img src="./utils/Ellipse15.png" alt="rectangle" id="ellipse15" />
+    </figure>
```

Comment:
- [ ] This must be a proper background image (decorations are always background images, not `img` tags)